modules/pouchdb/plugin.js

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue'
+import VuePouch from 'vue-pouch'
+import PouchDB from 'pouchdb'
+import Find from 'pouchdb-find'
+import LiveFind from 'pouchdb-live-find'
```

Comment:
Isn't necessary documenting of `find` and `live-find` existence with the module?